src/DevToys.PngCompressor/DevToys.PngCompressor.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCore)</TargetFrameworks>
+
+    <!-- NuGet package -->
+    <Version>1.0.0-pre.0</Version>
```

Comment:
The NuGet is ready to be deployed, but I didn't published it yet. I will do it as soon as this PR is completed.